activerecord/lib/active_record/base.rb

```diff
@@ -169,15 +169,15 @@ module ActiveRecord # :nodoc:
   #
   # Dynamic attribute-based finders are a mildly deprecated way of getting (and/or creating) objects
   # by simple queries without turning to SQL. They work by appending the name of an attribute
-  # to <tt>find_by_</tt> like <tt>Person.find_by_user_name</tt>.
+  # to +find_by_+ like +Person.find_by_user_name+.
   # Instead of writing <tt>Person.find_by(user_name: user_name)</tt>, you can use
   # <tt>Person.find_by_user_name(user_name)</tt>.
   #
   # It's possible to add an exclamation point (!) on the end of the dynamic finders to get them to raise an
   # ActiveRecord::RecordNotFound error if they do not return any records,
   # like <tt>Person.find_by_last_name!</tt>.
```

Comment:
```suggestion
  # like +Person.find_by_last_name!+.
```

Reply:
I think I did not include other characters in this expression, so it was missed